web/src/info_overlay.js

```diff
@@ -205,7 +205,7 @@ Coffee`,
                         <input type="checkbox" class="task" checked="checked">
                         <span></span>
                     </div>
-                    <strike><em><strong>Share draft budget</strong> - By Tuesday</em></strike>
+                    <del><em><strong>Share draft budget</strong> - By Tuesday</em></del>
```

Comment:
This change is wrong -- the HTML here is just a copy of the actual HTML produced by the `todo` widget, and that uses a `strike` tag for better or worse. 

We can change that detail; but a correct version of this change would edit `web/templates/widgets/todo_widget_tasks.hbs` and related todo widget code too.

Reply:
Changed `strike` tag to `del` in `web/templates/widgets/todo_widget_tasks.hbs` too.